### DISCUSSION TITLE
I am getting this error while sharing on GithHub in Intellije IDEA : Cannot load information for github.com/: Request response: Access to this site has been restricted. If you believe this is an error, please contact https://support.github.com.

### DISCUSSION THREAD

**Discussion by AtharvaBallal** [community/community#128045] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Error:
Cannot load information for github.com/: Request response: Access to this site has been restricted. If you believe this is an error, please contact https://support.github.com.

**Comment by ChunhThanhDe** [comment]
I have the same problem, it's a headache because of this 😢

**Reply by bisenankit21** [reply]
Update your IntelliJ Idea, it will work. Even I was facing this same issue and I updated, it worked for me.

**Reply by sandyds-uc** [reply]
Thank you. This fixed it for me.

**Reply by Overlord7667** [reply]
For those who are trying to upload a project from Android Studio to GitHub, you need to do the following:

1) Create a commit.
2) In the bottom left corner, click Git.
3)Left-click on the project to open the menu and then click Push.

**Reply by Hakkotako** [reply]
Update IntelliJ worked for me as well.

**Reply by yasiinio** [reply]
Yes, the update worked for me too.

**Reply by ramesh-aqfer** [reply]
yes. update fixed the issue for me too (Goland)

**Reply by WillTheCodeWork** [reply]
thanks, updating worked for me too

**Reply by RichStone** [reply]
Updating RubyMine from `2024.1.4` to `EAP 2024.2` worked for me.

**Reply by avallonazevedo** [reply]
+1 updating RubyMine fixes the issue.

**Reply by Manana80** [reply]
I have the same issue, my intelliJ is up to date. Still, I uninstalled it and reinstalled it .. I still can't share my project on github

**Reply by ChunhThanhDe** [reply]
> I have the same issue, my intelliJ is up to date. Still, I uninstalled it and reinstalled it .. I still can't share my project on github

after update to last version and restart computer , your fault still?

**Comment by anaijg** [comment]
Same problem here.

**Comment by thinun** [comment]
same i been looking for a solution all morning ;(

**Reply by ahad1105** [reply]
yes

**Comment by anaijg** [comment]
I only got to solve it manually, by creating an new repository in GitHub, and then using git commands in the terminal to upload the project.

**Reply by ChunhThanhDe** [reply]
If so, it seems the problem is not with Github but with Android Studio

**Reply by thinun** [reply]
This happened to me in webstorm

**Reply by anaijg** [reply]
And I was using IntelliJ IDEA with Java.

**Reply by Himanshu-Kandwal** [reply]
> If so, it seems the problem is not with Github but with Android Studio

I am also facing this problem in android studio

**Reply by Malik1110620** [reply]
I am also facing this problem in android studio

**Reply by komyelizqa** [reply]
I am facing it with IntelliJ IDEA

**Comment by vbatecan** [comment]
It's happening to me right now, any known fix? I was using Intellij IDEA with Java. I can't connect through Tasks

**Reply by MikiMilane1** [reply]
> It's happening to me right now, any known fix without creating a new repository? I was using Intellij IDEA with Java. I can't connect through Tasks

A new repo wont solve it. You cant push anything at the moment.

**Reply by vbatecan** [reply]
> > It's happening to me right now, any known fix without creating a new repository? I was using Intellij IDEA with Java. I can't connect through Tasks
> 
> A new repo wont solve it. You cant push anything at the moment.

Got it, my bad.

**Comment by Mrdudada** [comment]
Same problem! The gitee is allright but the github is'nt work, i am in china，does anyone know how to solve it?

**Reply by Mrdudada** [reply]
i sloved this problem! undate your Pycharm!

**Reply by DrenaGit** [reply]
Please, could you write more details. How can I undate my PyCharm? (I have the same problem with sharing my project from PyCharm on GitHub)

**Reply by Rk-D-prince** [reply]
bro did u find any solution

**Comment by vbatecan** [comment]
hopefully gets a fix soon.

**Reply by abhishekgiri93** [reply]
updating intellij worked for me

**Comment by grinsva** [comment]
Same problem! Now

**Comment by intellijh** [comment]
Try generating new token instead of the old one

**Reply by vbatecan** [reply]
Already tried that, no luck.

**Comment by zeroradish** [comment]
I have the same problem! Changing the token doesn't solve it... not only problems with IntelliJ tasks but also when using the GitHub API.

**Comment by Leykozz** [comment]
Update IDEA. It helped me

**Reply by MikiMilane1** [reply]
> Update IDEA. It helped me

Thanks. Worked for me as well

**Reply by strikanov** [reply]
not for me(

**Reply by DrBarns** [reply]
Updated Android studio from 2023.2 to 2023.3 (currently most recent) and it works now, thank you for the hint.

**Reply by mc738** [reply]
Thanks, this sorted the issue for me with Rider. Turns out there was an issue with Intellij products and GitHub which was solved with an update.

**Comment by DrBarns** [comment]
Same Problem with Android Studio Iguana 2023.2.1 Patch 2.
All new Token with all access granted, login is possible but no repository browsing and no sharing possible. 
Tried everything I could image, same error message.

**Comment by scientisthamster** [comment]
Just update your Android Studio, Jellyfish 2023.3.1 works for me fine

**Reply by saiful-softdev** [reply]
jellyfish 2023.3.1 
use same version same problem is here

**Comment by vbatecan** [comment]
Updating it works, seems like it is related to the security issue CVE-2024-37051 in JetBrains GitHub Plugin that could **compromise your GitHub Token** and was fixed in the latest update.

**Reply by justinmunro** [reply]
Updating PHP Storm as you said worked for me! Thanks!

**Comment by grinsva** [comment]
I have solved the problem. Updated Android Studio to the latest version

**Comment by codeviewigo** [comment]
I have the same problem with PyCharm. Could you please tell me how to solve this problem?

**Reply by justinmunro** [reply]
It seems to be to do with a security issue with JetBrains. I sorted it by just updating my PHPStorm to the latest version and then it worked. I would update PyCharm to the latest version too to see if that sorts it for you.

**Reply by codeviewigo** [reply]
Already updated it, but it didn't help (

**Reply by codeviewigo** [reply]
Turns out to have completely failed to update PyCharm. After installing all the patches, github worked again. Thanks!!!

**Reply by ghost** [reply]
I updated PyCharm and it did the work. Thanks

**Reply by josrdgv** [reply]
i already updated my pycharm but it didnt work 
for me

**Reply by Le-Cerveau-p** [reply]
> Turns out to have completely failed to update PyCharm. After installing all the patches, github worked again. Thanks!!!

what patches did you use please.

**Comment by Sonyque** [comment]
I had the same problem - token regeneration didn't help. Only after upgrading to the recent `WebStorm 2024.1.4` versions it started working.

**Comment by RoMANzhula** [comment]
For IntelliJ IDEA

Run IntelliJ IDEA -> Help -> Check for Updates... -> in the pop-up window click update and restart.
This helped me solve this problem.

Good luck to everyone.

**Reply by lxwang17283** [reply]
更新pycharm就行了

**Comment by tale80** [comment]
same problem on Rider, update fixed the issue.
Related to this problem : [Updates for security issue affecting IntelliJ-based IDEs 2023.1+ and JetBrains GitHub Plugin](https://blog.jetbrains.com/security/2024/06/updates-for-security-issue-affecting-intellij-based-ides-2023-1-and-github-plugin/)

**Reply by profK** [reply]
Updating rider didn't fix it for me :(
Nm update didnt complete...

**Reply by CristinaTRico** [reply]
Same updating did not fix

**Comment by AtharvaBallal** [comment, marked as answer]
Hello Guys, 

The Solution that works for me to solve the problem is reinstalling IntelliJ IDEA. You can also try to update the IDE.
@bisenankit21 Provided the solution to update IDE, Thank you.

Many people contributed and provided great ideas and Blog links to resolve this issue and help others. I want to thank all of you for your support. Thank you. 
I am closing the discussion and marking this as an answer this may work who are facing this issue if not then please go through all the comments.

**Reply by harry-929** [reply]
Updating the intellij worked for me 
Thanks :)

**Reply by ZougaRida** [reply]
For those of you like me can't see the popup to update to latest version for example in my case for Pycharm, all you need to do is head to help option up and then look check for updates, or click 2 times `Shift` key then type check for updates, hope it helps.

**Reply by pacesetter-avidal** [reply]
Was having the same issues, updated my intellij and it fixed my issue.  Thank y'all.

**Reply by victoriaborisovaa** [reply]
I have the same problem but the update didn't help. Any other solution?

**Reply by Polina153** [reply]
Thanks a lot, it helped me too

**Reply by ozturkcemal** [reply]
I had the same issue for PyCharm and the update worked for me as well.

**Comment by slawomirmarczynski** [comment]
Reinstall PyCharm

Pycharm Community 2024.1 (MS Windows 10, also MS Windows 11 on another computer) - the same problem, GitHub "access restricted".
I have uninstalled PyCharm 2024.1. BTW, some troubles, because uinstaller did not work. Thus I have deleted the entire PyCharm folder.
I have installed PyCharm 2024.1.3 (latest). Fixed, all work properlly.

**Comment by montaser101** [comment]
same problem with me any one fixed it?

**Reply by AtharvaBallal** [reply]
Please Could you check the Answer Tab? Otherwise try to update IDE.

**Comment by azizmarashi** [comment]
same problem with me

**Comment by AlexanderSparrow** [comment]
intelliJ IDEA 2024.1.3 (241.17890.1 build) Release Notes

Security fixes	Task	[CVE-2024-37051](https://www.cve.org/CVERecord?id=CVE-2024-37051)	Fixed security vulnerability related to JetBrains GitHub Plugin...

Updating  worked for me! Thanks!

**Comment by codepil0t** [comment]
same problem.... every update is a pain...

**Comment by profK** [comment]
If you are running JetBraons toolbox, it will have an Update button next to
it if an u[date is available.
Otherwise download the latest PyCharm for Jet Brains and install over your
current one.

On Sun, Jun 23, 2024 at 11:45 AM DrenaGit ***@***.***> wrote:

> Please, could you write more details. How can I undate my PyCharm? (I have
> the same problem with sharing my project from PyCharm on GitHub)
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/orgs/community/discussions/128045#discussioncomment-9852599>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AA64NZMBDCRFARPOE353ZE3ZI3US7AVCNFSM6AAAAABJCY6HW6VHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM4TQNJSGU4TS>
> .
> You are receiving this because you commented.Message ID:
> ***@***.***>
>


-- 
It's always darkest just before you are eaten by a grue.

**Comment by NadChel** [comment]
I wonder why invalidating VCS caches doesn't work. I don't feel like reinstalling the IDEA
```
IntelliJ IDEA 2024.1 (Community Edition)
Build #IC-241.14494.240, built on March 28, 2024
Runtime version: 17.0.10+8-b1207.12 amd64
VM: OpenJDK 64-Bit Server VM by JetBrains s.r.o.
Windows 10.0
GC: G1 Young Generation, G1 Old Generation
Memory: 2048M
Cores: 4
Registry:
  ide.experimental.ui=true
Non-Bundled Plugins:
  Lombook Plugin (241.14494.240)
  AntSupport (241.14494.158)
Kotlin: 241.14494.240-IJ
```

**Comment by profK** [comment]
worked for Rider too

On Mon, Jul 1, 2024 at 4:34 AM Harish Verma ***@***.***>
wrote:

> Updating the intellij worked for me
> Thanks :)
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/orgs/community/discussions/128045#discussioncomment-9923031>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AA64NZOGAQ46HJBY77VHCR3ZKEIBRAVCNFSM6AAAAABJCY6HW6VHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM4TSMRTGAZTC>
> .
> You are receiving this because you commented.Message ID:
> ***@***.***>
>


-- 
It's always darkest just before you are eaten by a grue.

**Comment by dev-michael-schmidt** [comment]
IntelliJ IDEA community:
Probably beating a dead horse here, but as others have said, updating the IDE and restarting it worked for me.  On Ubuntu (a derivative, Pop!_OS) I used the toolbox and updated to 2024.1.4

**Comment by IXBOB** [comment]
same probelm. updating the IDE helps me fix it. :)

**Comment by BobJackson** [comment]
update to latest fix the problem for my PyCharm.

**Comment by datatalking** [comment]
Updated my pycharm and worked immediately

**Comment by ZanogeaDumitru** [comment]
Updating works also for me on PHPStorm

**Comment by ahad1105** [comment]
I was using 2022 community edition and updating to latest version (i.e 2024 edition as of now) really solved the issue , thankyou

**Comment by 62mkv** [comment]
failed for me on IntelliJ 2023.2; works on 2024.2.3

**Comment by AyuEffortEdition** [comment]
actually my pycharm version is 2024.1, I think this version is new enough... Should I get update?

**Comment by ffelipecuevasc** [comment]
Same problem here using IntelliJ Ultimate with JAVA proyects and GitHub, any solution yet? Didn´t work for me updating IntelliJ.

**Reply by yanzhisishui** [reply]
samp problem now!!!! Is there no other way but to update idea？？？？

**Reply by AtharvaBallal** [reply]
@ffelipecuevasc  and @yanzhisishui 

Sorry for the late reply, can you guys please try to reinstall the latest version and let me know. Make sure to uninstall the previous and install the latest.

**Reply by ffelipecuevasc** [reply]
> @ffelipecuevasc and @yanzhisishui
> 
> Sorry for the late reply, can you guys please try to reinstall the latest version and let me know. Make sure to uninstall the previous and install the latest.

@AtharvaBallal I did that, didn't work. Problem still there.

**Reply by AtharvaBallal** [reply]
@ffelipecuevasc 
Did you import the previous settings after reinstalling. You can try one more thing after uninstalling IDE try to search intellije or your ide name on the computer and try to delete all the folders make sure to take back up all your projects and then try to install a fresh new version. Since there is no proper solution for this we have to try and find out by doing different approaches.

**Comment by z13636** [comment]
my pycharm version is 2024,1, Is it old?

**Reply by z13636** [reply]
or say it must be the most new version?

**Comment by ShiranK90** [comment]
I have faced the same issue in IntelliJ IDE and after updating to the latest version issue was resolved. 

1. Go to Help
2. Check for Updates...
3. Then after the Update IDE will restart and the try to Share the Project again

**Comment by AnryAnryK** [comment]
The Best solution is - using Console`s commands in IDE for Pushing to Git )

**Comment by putskan** [comment]
solved in pycharm by updating
